### PR TITLE
[wombat] fix warnings

### PR DESCRIPTION
**Pull request by spacey-sooty**
**What this PR does**  
Fixes warnings in Wombat

### CODE REVIEW

File: wombat/src/main/include/utils/Encoder.h [hunk 1]
```diff
@@ -31,7 +31,7 @@ namespace wom {
     units::radians_per_second_t GetEncoderAngularVelocity();   // rad/s
 
     int encoderType = 0;
-    double _reduction = 1.0;
+    double _reduction;
```
Comment (goanna247) [inline wombat/src/main/include/utils/Encoder.h:34]: not allowing a default reduction?

Comment (spacey-sooty) [reply wombat/src/main/include/utils/Encoder.h:34]: It didnt want it here as this initialises it before the constructor assigns it from the param, but if I put the default in the constructor it wants a default for everything

Comment (goanna247) [reply wombat/src/main/include/utils/Encoder.h:34]: do you want me to add it as an issue and you can solve it another time?

Comment (spacey-sooty) [reply wombat/src/main/include/utils/Encoder.h:34]: sounds good :+1:

**Review by goanna247 — APPROVED** [review]

**Pull request merged**